app/services/hyrax/collections/collection_member_query_service.rb

```diff
@@ -0,0 +1,91 @@
+module Hyrax
+  module Collections
+    # Responsible for retrieving collection members
+    class CollectionMemberQueryService
+      delegate :repository, to: :scope
+      attr_reader :scope, :params
+
+      # @param [Object] scope: Typically a controller object that responds to `repository`
```

Comment:
You can document this as `# @param scope [#repository] Typically the controller...`

Reply:
done.